api/Crt.Data/Repositories/UserRepository.cs

```diff
@@ -114,12 +114,16 @@ public async Task<PagedDto<UserSearchDto>> GetUsersAsync(string searchText, bool
                     : query.Where(u => u.EndDate != null && u.EndDate <= DateTime.Today);
             }
 
+            query = query.Include(u => u.CrtRegionUsers);
+
             var pagedEntity = await Page<CrtSystemUser, CrtSystemUser>(query, pageSize, pageNumber, orderBy, direction);
 
             var users = Mapper.Map<IEnumerable<UserSearchDto>>(pagedEntity.SourceList);
+            var userRegion = pagedEntity.SourceList.SelectMany(u => u.CrtRegionUsers).ToLookup(u => u.SystemUserId);
 
             foreach (var user in users)
             {
+                user.Regions = string.Join(",", userRegion[user.SystemUserId].Select(x => x.RegionId).OrderBy(x => x));
```

Comment:
RegionNumber instead of RegioonId since it's for display and users don't know anything about RegionId.